WindowsServerDocs/remote/remote-desktop-services/rds-vdi-recommendations-2004.md

```diff
@@ -0,0 +1,794 @@
+
+- Is the app current...does it need an update?
+- The app, if pinned to the Start Menu, might have live tile data to download
+- Does the app have a cache of data that needs to be updated, such as maps or weather?
+- Does the app have persistent data from the user's profile that needs to be presented at login (ex. Sticky Notes)
+ 
+With a default installation of Windows 10, not all UWP apps may be used by an organization. Therefore, if those apps are removed, there are less evaluations to take place…less caching, etc.  The second method here is to direct Windows to disable “consumer experiences”.  This reduces Store activity by having to check for every user what apps are installed, what apps are available, and then to start downloading some UWP apps.  The performance savings can be significant when there are hundreds or thousands of users, all start work at approximately the same time, or even starting work at rolling times across time zones.
+
+Connectivity and timing are important factors when it comes to UWP app cleanup. If you deploy your base image to a device with no network connectivity, Windows 10 cannot connect to the Microsoft Store and download apps and try to install them while you are trying to uninstall them. This might be a good strategy to allow you time to customize your image, and then update what remains at a later stage of the image creation process.
+
+If you modify your base .WIM that you use to install Windows 10 and remove unneeded UWP apps from the .WIM before you install, the apps will not be installed from the beginning and your subsequent profile creation times will be shorter.  There is a link later in this section with information on how to remove UWP apps from your installation .WIM file.
+
+A good strategy for the virtual desktop environment is to provision the apps you want in the base image, then limit or block access to the Microsoft Store afterward. Store apps are updated periodically in the background on normal computers. The UWP apps can be updated during the maintenance window when other updates are applied. There is a great paper on this topic, recently published, and available here:
+
+Microsoft: [Universal Windows Platform Apps](https://docs.citrix.com/en-us/xenapp-and-xendesktop/current-release/manage-deployment/applications-manage/universal-apps.html)
+
+#### Delete the payload of UWP apps
+
+UWP apps that are not needed are still in the file system consuming a small amount of disk space. For apps that will never be needed, the payload of unwanted UWP apps can be removed from the base image using PowerShell commands.  If you delete UWP app payloads out of the installation .WIM file using the links provided later in this section, you can start from the beginning with a very slim list of UWP apps.
+ 
+Run the following command to enumerate provisioned UWP apps from a running OS, as in this truncated example output from PowerShell:
+
+```powershell
+    Get-AppxProvisionedPackage -Online
```

Comment:
The indentation of the example text below is OK, but from my own point of view I would find it more practical to avoid extra spacing from a command I want to copy from the web page.
```suggestion
Get-AppxProvisionedPackage -Online
```
Feel free to disagree with this suggestion, if your experience is that this indentation is common and to be expected by most other users.

Reply:
Completely agree, and fixed the spacing